workflows/datalogger/machine_specific/create_excavator_bucket_poi_report_with_state/create_excavator_bucket_poi_report_with_state.py

```diff
@@ -190,87 +79,23 @@ def OutputLineObjects(a_file_ptr, a_machine_type, a_replicate, aux_control_data_
-            continue
-        Replicate.load_manifests(resource_config_processor, decoded_json['data']['manifest'])
-
-        aux_control_data_comp = FindAuxControlDataComponentInResourceConfiguration(a_resource_configuration=resource_config_processor)
-        aux_control_data_dict = GetAuxControlDataFromComponent(a_component=aux_control_data_comp)
+        ProcessReplicate(a_decoded_json=decoded_json, a_resource_config_dict=resource_definitions, a_assets_dict=assets, a_state_dict=state, a_resources_dir=resources_dir, a_report_file=report_file, a_header_list=header_list, a_server=server, a_site_id=args.site_id, a_headers=headers, a_machine_description_filter="Generic Excavator (3DMC)")
 
-        component_point_list = FindPointsOfInterestInResourceConfiguration(a_resource_configuration=resource_config_processor)
-        transform_component = FindTransformComponentInResourceConfiguration(a_resource_configuration=resource_config_processor)
-
-        object_list = []
-        if len(component_point_list) > 0:
-
-            for comp in component_point_list:
-
-                for point in comp["points"]:
-
-                    poi_local_space = GetPointOfInterestLocalSpace(a_point_of_interest_component=comp["points_component"], a_transform_component=transform_component, a_point_of_interest=point["point_node"])
-
-                    item_x = {
-                        "title" : point["display_name"] + " [x]",
-                        "value" : poi_local_space[1]
-                    }
-
-                    item_y = {
-                        "title" : point["display_name"] + " [y]",
-                        "value" : poi_local_space[0]
-                    }
-
-                    item_z = {
-                        "title" : point["display_name"] + " [z]",
-                        "value" : poi_local_space[2]
-                    }
-                    obj = {
-                        "items" : [item_x,item_y,item_z]
-                    }
-
-                    object_list.append(obj)
-
-        OutputLineObjects(report_file, resource_config_processor._json["description"], decoded_json, aux_control_data_dict, object_list, header_list, state[ac_uuid] if ac_uuid in state else {})
-
-logging.info("Writing report to {}".format(args.report_file_name))
+logging.info("Writing report to {}".format(args.datalogger_output_file_name))
 
 report_file.close()
 
 with open(report_file_name, 'r+') as report_file:
-   content = report_file.read()
-   report_file.seek(0)
-   report_file.write("Machine ID, Machine Type, Device ID, Machine Name, Time (UTC), GPS Mode, Error(H), Error(V), MC Mode, Reverse, Delay Id, Operator Id, Task Id, Surface Name")
-   for point_of_interest_name in header_list:
-    if point_of_interest_name == "lat" or point_of_interest_name == "lon" or point_of_interest_name == "alt" or point_of_interest_name == "dir":
-        report_file.write(", {}".format(point_of_interest_name))
-    else:
+    content = report_file.read()
```

Comment:
same here.. and possibly below if there are more instances of this pattern

Reply:
All instances updated.